src/app/clusters/pump-configuration-and-control-server/pump-configuration-and-control-server.cpp

```diff
@@ -27,12 +27,15 @@
 
 using namespace chip;
 
+
 void emberAfPumpConfigurationAndControlClusterServerInitCallback(EndpointId endpoint)
```

Comment:
Wouldn't it make more sense to add this callback (both here and in the helper.js bits) once the callback is actually needed for something?

Reply:
You could say that, but I thought is was ok to just add it up-front :-)

Reply:
Is it the correct way of adding generic PCC functionality/behavior?

Reply:
It's the right way to do things that need to happen at startup for all PCC implementations.  The question is whether we have anything like that.  :)